pkg/controllers/externalsecret/suite_test.go

```diff
@@ -90,7 +90,7 @@ var _ = BeforeSuite(func() {
 	Expect(err).ToNot(HaveOccurred())
 
 	err = (&Reconciler{
-		Client:                    k8sClient,
+		Client:                    k8sManager.GetClient(),
```

Comment:
Need to use the client from the manager or the controller cannot use the field index.

Reply:
I need to rethink on this 🙇  This may not be necessary

Reply:
Yes, this is correct. We have to pass the cache client or [this List call](https://github.com/external-secrets/external-secrets/blob/c73eee267afb8ea707b8b7e11be3adff01282822/pkg/controllers/externalsecret/externalsecret_controller.go#L684) does not work since the non-cache client does not have the index.